src/components/ServiceList.jsx

```diff
@@ -30,32 +65,76 @@ const ServiceList = () => {
     );
   }
 
-  return isLoading ? (
-    <p>Loading Baltimore County services...</p>
-  ) : (
-    <div>
-      <div style={flex}>
-        <i
-          className="fas fa-star dg_popular_star"
-          aria-hidden="true"
-          style={MostPopularServiceIconStyles}
-        ></i>
-        <p style={legendText}> - Indicates a Most Popular Service</p>
-      </div>
-      <div className="row">
-        <FilterList
-          items={serviceItems}
-          renderItem={props => (
-            <div
-              key={props.name.replace(/\s/, "-")}
-              className="d-flex col-lg-3 col-md-6 col-sm-6"
-            >
-              <ServiceIconLink {...props} />
+  let searchItemFound =
+    isFiltering === true && filteredItems.length === 0 ? false : true;
+
+  return (
+    <React.Fragment>
+      {isLoading ? (
+        <p>Loading Baltimore County services...</p>
+      ) : (
+        <div className="row">
+          <div className="col-md-3 col-xs-12">
+            <PopularityFilterCollapse
+              header="Categories"
+              id="Popular-filter"
+              onChange={handleIsPopularFilterChange}
+              checked={isMostPopular}
+              isExpanded={false}
+            />
+          </div>
+          <div className="col-md-9 col-xs-12">
+            <div>
```

Comment:
don't think this div tag is needed around the text input, but not sure.

Reply:
removed